logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/call/usecase/GetIncomingCallsUseCaseTest.kt

```diff
@@ -0,0 +1,78 @@
+package com.wire.kalium.logic.feature.call.usecase
+
+import app.cash.turbine.test
+import com.wire.kalium.logic.data.call.CallRepository
+import com.wire.kalium.logic.data.conversation.ConversationDetails
+import com.wire.kalium.logic.data.conversation.LegalHoldStatus
+import com.wire.kalium.logic.data.conversation.UserType
+import com.wire.kalium.logic.data.user.ConnectionState
+import com.wire.kalium.logic.feature.call.Call
+import com.wire.kalium.logic.feature.call.CallStatus
+import com.wire.kalium.logic.framework.TestConversation
+import com.wire.kalium.logic.framework.TestUser
+import com.wire.kalium.logic.sync.SyncManager
+import io.mockative.Mock
+import io.mockative.classOf
+import io.mockative.given
+import io.mockative.mock
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.test.runTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class GetIncomingCallsUseCaseTest {
+
+    @Mock
+    val callRepository: CallRepository = mock(classOf<CallRepository>())
+
+    @Mock
+    val syncManager: SyncManager = mock(classOf<SyncManager>())
+
+    private lateinit var getIncomingCallsUseCase: GetIncomingCallsUseCase
+
+    @BeforeTest
+    fun setUp() {
+        getIncomingCallsUseCase = GetIncomingCallsUseCaseImpl(callRepository, syncManager)
+    }
+
+    @Test
+    fun givenEmptyCallList_thenEmptyNotificationList() = runTest {
```

Comment:
This is a use case for getting incoming calls, it has nothing related to notification.
Please follow given..when..then pattern for test cases

`givenAnEmptyCallList_whenInvokingGetIncomingCallsUseCase_thenEmitsAnEmptyListOfCalls`